config/global.php

```diff
@@ -66,4 +66,10 @@
 
     'Piwik\EventDispatcher' => DI\object()->constructorParameter('observers', DI\get('observers.global')),
 
+    'Zend_Validate_EmailAddress' => function () {
+        return new \Zend_Validate_EmailAddress(array(
+            'hostname' => new \Zend_Validate_Hostname(array(
+                'tld' => false,
```

Comment:
I tested all available settings, and I think `tld=false` is the only one we need to use. There was a problem with out-dated list of TLD zones that Zend used to validate email.